Makefile.am

```diff
@@ -85,3 +86,9 @@ desktopdir = $(datadir)/applications
 desktop_in_files = geany.desktop
 desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
 @INTLTOOL_DESKTOP_RULE@
+
+appdata_in_files = geany.appdata.xml.in
+
+appdatadir = $(datadir)/appdata
```

Comment:
Since its for distros why should we say where it goes?  Different distributions put things in different places, it shoudlbe up to them.  Since this is no use for anyone who installs geany from tarball or source it doesn't need to be installed by our build.